plugin/powerControl/powerControl.yaml

```
@@ -0,0 +1,43 @@
+  Org: "YOUR_ORG"
+  Bucket: "YOUR_BUCKET"
+
+IPMI:
+  User: "YOUR_IPMI_USER"                       
+  Password: "YOUR_IPMI_PASSWORD"
+  MaxNodesPerBatch: 10       
+  BatchIntervalSeconds: 60    
+  NodeBMCMapping:                 
+    craned1: "10.129.227.172"
+    craned2: "10.129.227.189"
+  ExcludeNodes:                    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)